src/SignhostAPIClient/Rest/DataObjects/Transaction.cs

```diff
@@ -7,9 +7,11 @@ public class Transaction
 	{
 		public string Id { get; set; }
 
+		public IDictionary<string, FileEntry> Files { get; set; }
+
 		public int Status { get; set; }
 
-		public bool Seal { get; set; }
+		public bool? Seal { get; set; }
```

Comment:
Although it is optional in the API due to backwards compability reasons it is better to make this more explicit in the client libraries and not make this an optional property.

Also this change would require a new major nuget version.